src/number-formatting/format.js

```diff
@@ -22,7 +53,8 @@ function formatNegativeNumber(str, options) {
 function expandNumber(number, precision, groupSizes, sep, decimalChar) {
 	var curSize = groupSizes[0],
 			curGroupIndex = 1,
-			numberString = String(number),
+			decimalNumber = convertScientificToDecimal(number),
+			numberString = String(decimalNumber),
```

Comment:
wouldn't it be clearer to rename convertScientificToDecimal to numberToString() and use it directly to get numberString ?